zeebe/logstreams/src/main/java/io/camunda/zeebe/logstreams/impl/flowcontrol/AppendMetricsDoc.java

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.logstreams.impl.flowcontrol;
+
+import io.camunda.zeebe.util.micrometer.ExtendedMeterDocumentation;
+import io.micrometer.common.docs.KeyName;
+import io.micrometer.core.instrument.Meter.Type;
+
+/** Flow control metrics for the log storage appender. */
+@SuppressWarnings("NullableProblems")
+public enum AppendMetricsDoc implements ExtendedMeterDocumentation {
+  /** Number of deferred appends due to backpressure */
+  TOTAL_DEFERRED_APPEND_COUNT {
```

Comment:
:wrench:  should we add the partition to the KeyName docs ?
Even though it comes from the registry, it's a field in the meter that should be documented.
I think we forgot to add it in the past few meters, but it's probably better to add it?

Reply:
Yeah, good point.